rc/windowing/zellij.kak

```diff
@@ -0,0 +1,69 @@
+# https://zellij.dev/
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+provide-module zellij %{
+
+# ensure we're running under zellij
+evaluate-commands %sh{
+    [ -z "${client_opt_windowing_modules}" ] || [ -n "$ZELLIJ" ] || echo 'fail zellij not detected'
+}
+
+declare-option -hidden -docstring %{zellij run options (
+    -c, --close-on-exit            Close the pane immediately when its command exits
+        --cwd <CWD>                Change the working directory of the new pane
+    -d, --direction <DIRECTION>    Direction to open the new pane in
+    -f, --floating                 Open the new pane in floating mode
+    -h, --help                     Print help information
+    -n, --name <NAME>              Name of the new pane
+    -s, --start-suspended          Start the command suspended, only running after you first presses ENTER)}\
+    str zellij_run_options ""
+declare-option -hidden -docstring %{zellij run command} str zellij_run_cmd ""
+
+define-command -hidden -params 0 zellij-run %{
+    nop %sh{
+    zellij run ${kak_opt_zellij_run_options} -- $SHELL -i -c "${kak_opt_zellij_run_cmd}" < /dev/null > /dev/null 2>&1 &
+    }
+    set global zellij_run_options ""
+    set global zellij_run_cmd ""
+}
+
+define-command -hidden -params 1.. zellij-terminal-impl %{
+    set global zellij_run_cmd "%arg{@}"
+    zellij-run
+}
+complete-command zellij-terminal-impl shell
+
+define-command zellij-terminal-vertical -params 1.. -docstring '
+zellij-terminal-vertical <program> [<arguments>]: create a new terminal as a zellij pane
+The current pane is split into two, top and bottom
+The program passed as argument will be executed in the new terminal' \
+%{
+    set global zellij_run_options "--direction down"
+    set global zellij_run_cmd "%arg{@}"
+    zellij-run
+}
+complete-command zellij-terminal-vertical shell
+
+define-command zellij-terminal-horizontal -params 1.. -docstring '
+zellij-terminal-horizontal <program> [<arguments>]: create a new terminal as a zellij pane
+The current pane is split into two, left and right
+The program passed as argument will be executed in the new terminal' \
+%{
+    set global zellij_run_options "--direction right"
+    set global zellij_run_cmd "%arg{@}"
+    zellij-run
+}
+complete-command zellij-terminal-horizontal shell
+
+define-command zellij-focus -params 0 -docstring '
```

Comment:
`focus` alias is invoked every once in a while and should not be interactive, right @mawww ?

Reply:
let me check other focus implementations, will follow the standards, thanks

Reply:
Fixed it, thanks for the review!